bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/EscalationEventDefinitionValidator.java

```diff
@@ -34,14 +38,26 @@ public Class<EscalationEventDefinition> getElementType() {
   public void validate(
       final EscalationEventDefinition element,
       final ValidationResultCollector validationResultCollector) {
+    final Escalation escalation = element.getEscalation();
+    final ModelElementInstance parentElement = element.getParentElement();
 
-    if (isEscalationThrowEvent(element) && element.getEscalation() == null) {
+    if (isEscalationThrowEvent(parentElement) && escalation == null) {
       validationResultCollector.addError(0, "Must reference an escalation");
     }
+
+    if (parentElement instanceof CatchEvent && escalation != null) {
+      final String escalationCode = escalation.getEscalationCode();
+      if (escalationCode != null && escalationCode.startsWith(ZEEBE_EXPRESSION_PREFIX)) {
+        validationResultCollector.addError(
+            0,
+            String.format(
+                "Expected static value of '%s', but was: '%s'",
+                ((CatchEvent) element.getParentElement()).getId(), escalationCode));
```

Comment:
🔧 Because we moved this to the `EscalationEventDefinitionValidator` the relevant element will be appended automatically. So we could consider changing the error to `EscalationCode may not be an expression`.

I can also live with this message, just a thought.

Reply:
@remcowesterhoud. Let's be consistent with the error catch event. 

> The escalationCode of the escalation catch event is not allowed to be an expression

WDYT?

Reply:
Fine with me 👍